.changeset/small-dingos-grin.md

```diff
@@ -0,0 +1,5 @@
+---
+"@blockprotocol/type-system": patch
+---
+
+add stdlib functions for interacting with subgraph. reorganize types
```

Comment:
Shouldn't this be the changelog for the graph package?

I think the changes we made in the type-system were a variety of updates to the schema representation such as "additionalProperties" or removing redundant fields on entities such as "requiredLinks"

Reply:
```suggestion
update schema representations of ontology types to remove redundant fields, and add further validation
```